mix.exs

```diff
@@ -93,17 +96,14 @@ defmodule Tesla.Mixfile do
       source_url: @source_url,
       source_ref: "v#{@version}",
       skip_undefined_reference_warnings_on: [
-        "CHANGELOG.md",
         "guides/howtos/migrations/v1-macro-migration.md"
       ],
       extra_section: "GUIDES",
       logo: "guides/elixir-tesla-logo.png",
       extras:
         [
           "README.md",
-          "LICENSE"
-          # TODO: add CHANGELOG.md
-          # "CHANGELOG.md"
```

Comment:
we will need this changelog after https://github.com/elixir-tesla/tesla/pull/728